src/main/java/Duke.java

```diff
@@ -1,10 +1,56 @@
+import ui.Ui;
+import java.util.*;
+
 public class Duke {
+    protected static List<String> tasks = new ArrayList<String>();
+
+    public static void printList() {
+        for (int i = 0; i < tasks.size(); i++) {
+            System.out.println(i+1 + ". " + tasks.get(i));
+        }
+    }
+
     public static void main(String[] args) {
+        boolean isExit = false;
+
+
+        /*
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+
+         */
+
+        Ui ui = new Ui();
+        ui.welcome();
+        while (!isExit) {
+            String command = ui.readCommand();
+
+            switch(command) {
+                case "bye":
```

Comment:
A small suggestion, maybe there is no indentation for case clauses in the switch statement.